src/Engine/Audio/SteamAudioInterface.hpp

```diff
@@ -1,8 +1,10 @@
 #pragma once
+
 #include <phonon.h>
 #include <vector>
 #include <cstdint>
 #include "SteamAudio.hpp"
+#include "../linking.hpp"
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'SteamAudioInterface' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)